.gitignore

```diff
@@ -1,2 +1,3 @@
 dist/
 node_modules/
+.DS_Store
```

Comment:
I think there's a carriage return missing here. (I know, it's a bit pedantic)

Reply:
As a pedant myself, I have no qualms with fixing this

Reply:
Missing a period in your comment.